lib/custom_wizard/field.rb

```diff
@@ -3,14 +3,17 @@ def self.types
     @types ||= {
       text: {
         min_length: nil,
+        max_length:nil,
         prefill: nil
       },
       textarea: {
         min_length: nil,
+        max_length:nil,
```

Comment:
@fzngagan Let's get the spacing on this right please.

Reply:
Done